Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.21 as builder
 WORKDIR /go/src/github.com/openshift/cli-manager
 COPY . .
 RUN make build --warn-undefined-variables
 
-FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
+FROM registry.access.redhat.com/ubi9:latest
```

Comment:
This will not work as we need the fips compliant image

Reply:
Thanks. I've updated with `http://registry.redhat.io/rhel9-2-els/rhel:9.2-1222`. Good thing is that I've already imported a secret for registry.redhat.io. This image would work better than ubi9.